watertap/unit_models/electroNP_ZO.py

```diff
@@ -119,66 +71,18 @@ def _process_config(self):
                 "The ElectroNP model was expecting at least one solute or ion and did not receive any."
             )
 
-    def build(self):
-        # Call UnitModel.build to setup dynamics
-        super().build()
+        if "treated" and "byproduct" not in self.config.outlet_list:
+            raise ConfigurationError(
+                "{} encountered unrecognised "
+                "outlet_list. This should not "
+                "occur - please use overflow "
+                "and underflow as outlets.".format(self.name)
```

Comment:
This ConfigurationError should say "treated" and "byproduct" instead of "overflow" and "underflow", right?

Reply:
yes